src/test/java/parser/ParserTest.java

```diff
@@ -0,0 +1,71 @@
+package parser;
+
+import command.*;
```

Comment:
Perhaps can list out all the import statements explicitly?

Reply:
Great Point! Will push this out in the next commits!